app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,37 @@
+class LikesController < ApplicationController
+  before_action :find_event
+  before_action :find_like, only: [:destroy]
+
+  def create
+    if already_liked?
+      flash[:notice] = "You can't like more than once"
+    else
+      @event.likes.create(user_id: current_user.id)
+    end
+    redirect_to event_path(@event)
+  end
+
+  def destroy
+    if !(already_liked?)
+      flash[:notice] = "Cannot unlike"
```

Comment:
Hmm I think if you already liked an event, and you press Like again, I would expect only to remove the like, like a boolean.

So I think yes, a user can unlike

Reply:
So in this case maybe the message should be "You are trying to unlike, but no like is preset, Do you want to like the event instead?"

Reply:
with like, it's to avoid that a user can like more than once on a single event.

Reply:
Here I am proposing change of the message, what do you think of the change?

Reply:
that a good idea

Reply:
I made the new push and made the correction